src/python/pants/notes/1.12.x.rst

```diff
@@ -0,0 +1,283 @@
+1.12.x Stable Releases
+======================
+This document describes releases leading up to the ``1.12.x`` ``stable`` series.
+
+
+1.12.0rc0 (11/19/2018)
+----------------------
+
+New features
+~~~~~~~~~~~~
+
+* Add prelude and epilogue (#6784)
+  `PR #6784 <https://github.com/pantsbuild/pants/pull/6784>`_
+
+Bugfixes
+~~~~~~~~
+
+* Use ThreadPool for cache fetching and rust tar for artifact extration (#6748)
```

Comment:
extraction

Reply:
I used to do this, but once its here via the changelog helpers, its committed in perpetua in git history so I don;t think there is uch point in either reviewing or fixing commit message typos only in the changelog.

Reply:
I can see that change log aims to be more human friendly, but yeah, I see your point. Now that's corrected, there's no reason to make it a typo again.